README.md

```diff
@@ -5,14 +5,17 @@
 
 ![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
 
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary><b> Note for Hacktoberfest </b></summary>
```

Comment:
Do you think the header would look fine? I think it'd be nice.
```suggestion
  <summary><h2> Note for Hacktoberfest <h2></summary>
```

Reply:
Unfortunately, adding a header tag would push the content to the next line and the dropdown arrow would be above it. Here's the screenshot: 

![image](https://user-images.githubusercontent.com/43907522/145723816-ac8cc557-1909-443e-ab03-7d0cd2a8884f.png)

However, we can achieve the same effect of a header tag by including some inline styling. Here's what that would look like:

![image](https://user-images.githubusercontent.com/43907522/145723966-af106540-605c-40fe-a08e-621cd6e5b73a.png)

Please let me know which one you would prefer and I'll make the changes accordingly.

Reply:
The second option looks good. 👍

Reply:
The second option looks good. 👍

Reply:
@arghyab0 @Panquesito7 now it has been committed, in the readme the styling to make it into a header has not been effective, as it still looks like a paragraph text.

Reply:
@CodingSpecies I just checked and it seems that GitHub sanitises the HTML and removes inline styling amongst other things. 
We could do something like this: 

![image](https://user-images.githubusercontent.com/43907522/145753909-5199102a-c4f3-4a9d-85c1-de4a2677c4c4.png)

Another option would be to have a separate MD file and link it to the heading. Here's what that would look like:

![image](https://user-images.githubusercontent.com/43907522/145754066-2663e75b-3f3e-4c8d-9be1-d4cbf1404842.png)